include/functions.h

```diff
@@ -2718,8 +2718,8 @@ UNK_TYPE func_8013D68C(Path* path, s16 arg1, UNK_PTR arg2);
 // void func_8013D720(void);
 UNK_TYPE func_8013D768(Actor* actor, UNK_PTR arg1, s16 arg2);
 // void func_8013D83C(void);
-// void func_8013D8DC(void);
-// void func_8013D924(void);
+s32 func_8013D8DC(s8 arg0, GlobalContext* globalCtx);
+s8 func_8013D924(s32 arg0, GlobalContext* globalCtx);
 Actor* func_ActorCategoryIterateById(GlobalContext* globalCtx, struct Actor* actorListStart, s32 actorCategory, s32 actorId);
```

Comment:
```suggestion
s8 func_8013D924(s32 arg0, GlobalContext* globalCtx);
Actor* func_ActorCategoryIterateById(GlobalContext* globalCtx, Actor* actorListStart, s32 actorCategory, s32 actorId);
```
I know this is outside your PR but I can't help myself lol